pkgs/tools/wayland/swww/default.nix

```diff
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, fetchFromGitHub, rustPlatform, pkg-config, lz4
+, libxkbcommon, ... }:
+rustPlatform.buildRustPackage rec {
+  pname = "swww";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "Horus645";
+    repo = pname;
+    rev = "v0.7.0";
+    sha256 = "ofJTL+9izMmwENaEVYQ7rjpoEFR1J2szkJxzYb09N7g=";
+  };
+  cargoSha256 = "sha256-ntulNQUQZ42CGvwnnQSMQsr/MDSNY/5/Aq2Ew9QItn8=";
+  buildInputs = [ lz4 libxkbcommon ];
+  doCheck = false; # Integration tests do not work in sandbox enviroment
```

Comment:
Have you tried to resolve the test failures? It looks like maybe it's just a path issue.

```
stderr=```"Error: \"error spawning swww-daemon: No such file or directory (os error 2)\"\n"```
command=`"/build/source/target/x86_64-unknown-linux-gnu/release/swww" "init"`
code=1
```

Reply:
I have not. I remember asking about this on discord when I first tried to build swww. Somebody pointed out that the issue is that the integration tests use the local network and are doomed to fail; see [https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md?plain=1#L331](documentation here). That said, I am not well-versed in nix or rust. So if you have an idea to solve it, I am happy to try it.

Reply:
It also looks like the tests check for a functioning wayland environment, so probably best to disable them as you did here.

```
thread \'main\' panicked at \'Failed to connect to wayland environment: NoCompositorListening\', daemon/src/wayland.rs:51:45
```